dc/council/code/titles/13/sections/13-339.xml

```diff
@@ -3,14 +3,14 @@
   <num>13-339</num>
   <heading>Form of order of publication.</heading>
   <text>An order of publication shall be in the following or an equivalent form:</text>
-  <para>
-    <heading>United States District Court for the District of Columbia.</heading>
-    <text>AB, plaintiff,</text>
-    <text>versus          In ____. No. ____</text>
-    <text>CD, defendant.</text>
-    <text>The object of this action is to (state it briefly).</text>
-    <text>On motion of the plaintiff, it is this ____ day of ____, A.D. ____, ordered that the defendant cause his appearance to be entered herein on or before the fortieth day, exclusive of Sundays and legal holidays, occurring after the day of the first publication of this order; otherwise the cause will be proceeded with as in cause of default. ________ Judge.</text>
-  </para>
+  <text>
+  <strong>United States District Court for the District of Columbia.</strong>
+  <br/>
```

Comment:
Same question as 1-204.63. Why use a <br/> instead of closing the </text>, and then reopening for "AB, ....".